src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.MoveFile.cs

```diff
@@ -0,0 +1,238 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Threading;
+using Xunit;
+
+public partial class FileSystemWatcher_4000_Tests
+{
+    #region WindowsTests
+
+    [Theory]
+    [InlineData(WatcherChangeTypes.Changed, "changed", false)]
+    [InlineData(WatcherChangeTypes.Created, "created", false)]
+    [InlineData(WatcherChangeTypes.Deleted, "deleted", false)]
+    [InlineData(WatcherChangeTypes.Deleted, "changed", false)]
+    [InlineData(WatcherChangeTypes.Renamed, "renamed", true)]
+    [InlineData(WatcherChangeTypes.Renamed, "changed", true)]
```

Comment:
It was on purpose to match the old tests that were there. I wasn't too familiar with the inner workings of the FSW so I didn't want to risk removing any coverage or realistic test cases by changing them too much. 

There were several cases where they listened for one thing and then tested for a different thing.